workshops/discord_message_bot/README.md

```diff
@@ -0,0 +1,395 @@
+---
+name: 'Discord Custom Message Bot'
+description: 'Save your own own custom messages with a Discord Bot!'
+author: '@JakeGerber'
+---
+
+# Create a Discord Bot that allows you to save custom messages!
+
+<img src="https://cloud-bj4vorj8t.vercel.app/examplebot.png" width="380" alt="Message Example">
+
+## Bot Setup
+
+Let's set up a bot through Discord before creating its features! Click [here](https://discord.com/developers/applications).
+
+Click the "New Application" button at the top right.
+<img src="https://cloud-noinx6mnk.vercel.app/0screenshot__1379_.png" width="900" alt="New Application Button">
+
+This creates a new bot where you can customize the name, description, and profile picture!
+<img src="https://cloud-jlzsnh85x.vercel.app/0screenshot__1380_.png" width="900" alt="Bot Profile">
+
+Now click the "Bot" tab on the right side of your screen.
+<img src="https://cloud-e5r0obhgo.vercel.app/0screenshot__1381_.png" width="900" alt="Bot Side Bar">
+
+Click "Add Bot" to generate a bot token! This identifies the bot! Give it to nobody!
```

Comment:
I think you should remove the first 2 exclamation marks, only keep the last one. Right now this reads awkwardly